### ISSUE TITLE
WCAG 2.1: Reflow - Help text in search box is not completely available at 200% zoom

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#769]
**Describe the bug**
The search box help text in the left pane in "UI Automation tree view" is not completely accessible when the pane splitter is in default position (Considering the splitter position in Automated Checks as default)

**To Reproduce**
Adjust the following display settings in Windows :
- Display resolution to ```1280 * 1024```
- Scaling size to ```200%```

1. Launch AI-Windows
2. Run test on any app
3. Navigate to "View results in UI Automation Tree"
4. Set the pane splitter to a similar position as in automated checks
5. Observe that the complete placeholder help text is not available

**Expected behavior**
Help content should be made available i.e. completely visible or be revealed via accessible controls

**Actual behavior**
Help Text content  is hidden

**Screenshots or .GIF**
![5](https://user-images.githubusercontent.com/22068434/80000951-aa81e880-8472-11ea-8c7b-61c68a3cc48d.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise, Version1903
 - Accessibility Insights for Windows Version: Production Release: v1.1.1198.01

**Comment by adarshrema** [comment]
From internal testing for accessibility. To be combined with results from planned HC testing.

**Comment by juchampa** [comment]
@iamrafan This item is not in scope
